example/lib/stage_data/my_list_tile_widget_stage.dart

```diff
@@ -0,0 +1,126 @@
+import 'package:flutter/material.dart';
+import 'package:widget_stage/widget_stage.dart';
+
+class MyListTileWidgetStage extends WidgetStageData {
+  MyListTileWidgetStage()
+      : _tileCount = IntFieldConfigurator(
+          value: 1,
+          name: 'tileCount',
+          type: FieldConfiguratorType.stage,
+        ),
+        _listPadding = PaddingFieldConfigurator(
+          value: EdgeInsets.zero,
+          name: 'listPadding',
+          type: FieldConfiguratorType.stage,
+        ),
+        _title = StringFieldConfigurator(value: 'My List Tile', name: 'title'),
+        _stageColor = ColorFieldConfigurator(
+          value: Colors.transparent,
+          name: 'stageColor',
+          type: FieldConfiguratorType.stage,
+        ),
+        _circleColor = ColorFieldConfigurator(value: Colors.purple, name: 'circleColor'),
+        _tileColor = ColorFieldConfiguratorNullable(value: Colors.cyan, name: 'tileColor'),
+        _textColor = ColorFieldConfiguratorNullable(value: Colors.white, name: 'textColor'),
+        _borderRadius = DoubleFieldConfiguratorNullable(value: 10, name: 'borderRadius'),
+        _tileGap = DoubleFieldConfigurator(
+          value: 0,
+          name: 'tileSpace',
+          type: FieldConfiguratorType.stage,
+        );
+
+  @override
+  List<FieldConfigurator> get fieldConfigurators {
+    return [
+      _tileCount,
+      _listPadding,
+      _title,
+      _tileGap,
+      _stageColor,
+      _circleColor,
+      _tileColor,
+      _borderRadius,
+      _textColor,
+    ];
+  }
+
+  @override
+  String get name => 'MyListTileWidget';
+
+  final IntFieldConfigurator _tileCount;
+  final DoubleFieldConfigurator _tileGap;
+  final DoubleFieldConfiguratorNullable _borderRadius;
+  final PaddingFieldConfigurator _listPadding;
+  final StringFieldConfigurator _title;
+  final ColorFieldConfigurator _stageColor;
+  final ColorFieldConfigurator _circleColor;
+  final ColorFieldConfiguratorNullable _textColor;
+  final ColorFieldConfiguratorNullable _tileColor;
+
+  @override
+  Widget widgetBuilder(BuildContext context) {
+    return ColoredBox(
+      color: _stageColor.value,
+      child: ListView.builder(
+        padding: _listPadding.value,
+        itemCount: _tileCount.value,
+        itemBuilder: (context, index) {
+          return Padding(
+            padding: EdgeInsets.only(bottom: _tileGap.value),
+            child: _MyTitleTileWidget(
```

Comment:
If you want, you can use a `ListView.separated`, as it inserts separators between the item. This way, the last item would not get a bottom padding 😃 

```suggestion
      child: ListView.separated(
        padding: _listPadding.value,
        itemCount: _tileCount.value,
        separatorBuilder: (_, __) => SizedBox(height: _tileGap.value),
        itemBuilder: (context, index) {
          return _MyTitleTileWidget(
```